app/lib/theme/CustomDiffFields.tsx

```diff
@@ -55,13 +109,19 @@ const showStringDiff = (
         {isDate ? getLocaleFormattedDate(newData) : newData}
       </span>
       {contentSuffix && contentSuffixElement(id, contentSuffix)}
+      {/* Ultimately, we'll only show the latest committed value if there's a conflict, but show for now to facilitate manual testing. */}
```

Comment:
I'm assuming this comment applies to the rest of the commented out lines throughout this file. I'm fine with leaving them in for reference for future work.

Reply:
The comment does apply to the rest of the commented lines, yes